solvation_analysis/tests/conftest.py

```diff
@@ -99,6 +101,21 @@ def atom_groups(u_real):
     return atom_groups
 
 
+def test_temporary(solvation_data_large):
+    path = "/Users/orioncohen/projects/lowt/data/for_lammps_EL_production_BN_EC/BN10_EC_LiPF6"
```

Comment:
That should not be in production code...

Reply:
Thanks for the catch! Comments are always welcome.

I'm going to eliminate this test before merging. I just needed temporary access to a longer trajectory so that I could benchmark the speed of my implementation.

More generally, this is a very early stage PR. I'll definitely add thorough documentation and testing before merging it.